### DISCUSSION TITLE
WiX Toolset 5.0.0-rc.1 producing Deutsch msi

### DISCUSSION THREAD

**Discussion by jcasale** [wixtoolset/issues#8034] [Questions]
I cloned a repo of mine and updated the build action to use 5.0.0-rc.1 so I could start getting up to speed.

Although my [Package](https://github.com/jcasale/vim-msi-dev/blob/187f2ec9eeed4f4d5892b414da01bb8f31bdc66c/src/Product.wxs#L6) element sets the `Language` attribute to 1033, the result of this msi appears to be Deutsch?

I built some simpler msi's for different applications and they didn't reproduce this problem.

Any idea why this might be happening to the source in the link provided?

**Comment by barnson** [comment]
That's officially weird. Please open a bug.

**Reply by jcasale** [reply]
Issue opened at [8036](https://github.com/wixtoolset/issues/issues/8036).

Thanks.

**Reply by jcasale** [reply]
Hi @barnson Could the localization issue resolved in the associated issue lead to msi error code 2707 when using the wix built-in harvesting functionality?

**Reply by barnson** [reply]
:shrug: Seems unlikely.
